wandb/integration/catboost/catboost.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+catboost init
+"""
+
+import wandb
+
+
+class WandbCallback:
+    """`WandbCallback` automatically integrates CatBoost with wandb.
+
+    Arguments:
+        - metric_period: (int) if you are passing `metric_period` to your CatBoost model please pass the same value here (default=1).
+
+    Passing `WandbCallback` to CatBoost will:
+    - log training and validation metrics at every `metric_period`
+    - log iteration at every `metric_period`
+
+    Example:
+        ```
+        train_pool = Pool(train[features], label=train['label'], cat_features=cat_features)
+        test_pool = Pool(test[features], label=test['label'], cat_features=cat_features)
+
+        model = CatBoostRegressor(iterations=100,
```

Comment:
nit, maybe move the `iterations` argument to a new line

Reply:
👍🏻 In general, please re-format the example code. I'd copy it into a dummy file, the run `tox -e format -- dummy.py` and copy back into docstrings.